src/contrib/persistence/Akka.Persistence.Sql.TestKit/SqlJournalConnectionFailureSpec.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Persistence.Sql.TestKit
+{
+    public abstract class SqlJournalConnectionFailureSpec : Akka.TestKit.Xunit2.TestKit
+    {
+        protected static readonly string DefaultInvalidConnectionString = "INVALID_CONNECTION_STRING";
+
+        public SqlJournalConnectionFailureSpec(Config config = null, ITestOutputHelper output = null)
+            : base(config)
+        {
+        }
+
+        [Fact]
+        public void Persistent_actor_should_throw_exception_upon_connection_failure()
+        {
+            EventFilter.Exception<Exception>().ExpectOne(() =>
```

Comment:
The test expects one `Exception` to be thrown when the `PersistenceActor` is sent a message that triggers `Persist`.  Since the `PersistenceActor` has been given a bad connection string, it should throw an exception that it cannot initialize because it can't create a new `DbConnection`